src/twisted/conch/test/keydata.py

```diff
@@ -84,6 +85,16 @@
   'curve': b'ecdsa-sha2-nistp521'
 }
 
+ED25519Data = {
```

Comment:
I don't have a strong opinion on whether or not this should be done, but I think it'd be best to do it as it's own ticket.